src/packages/block/block-grid/property-editors/block-grid-group-configuration/property-editor-ui-block-grid-group-configuration.element.ts

```diff
@@ -0,0 +1,61 @@
+import { html, customElement, property, css } from '@umbraco-cms/backoffice/external/lit';
+import { UmbPropertyEditorUiElement } from '@umbraco-cms/backoffice/extension-registry';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import type { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UmbId } from '@umbraco-cms/backoffice/id';
+import { BlockGridGroupConfigration } from '@umbraco-cms/backoffice/block';
+
+@customElement('umb-property-editor-ui-block-grid-group-configuration')
+export class UmbPropertyEditorUIBlockGridGroupConfigurationElement
+	extends UmbLitElement
+	implements UmbPropertyEditorUiElement
+{
+	private _value: Array<BlockGridGroupConfigration> = [];
+
+	@property({ type: Array })
+	public get value(): Array<BlockGridGroupConfigration> {
+		return this._value;
+	}
+	public set value(value: Array<BlockGridGroupConfigration>) {
+		this._value = value || [];
+	}
+
+	@property({ attribute: false })
+	public set config(config: UmbPropertyEditorConfigCollection | undefined) {}
+
+	#addGroup() {
+		if (Array.isArray(this.value)) {
```

Comment:
If you used _value here, then you can trust that its always an array.. Since you initially set it to an array And you as well ensure its an array as part of the value setter.